rom-hacking/scripting/animation-commands/ac-world-x.md

```diff
@@ -0,0 +1,34 @@
+# (786) _AC_WORLD_X
```

Comment:
Both this file and ac-world-z need to be .mdx files for the blurbs to work properly.